.coderabbit.yaml

```
@@ -0,0 +1,240 @@
+# AI-powered code review and analysis configuration
+
+# Reviews configuration
+reviews:
+  # Enable review generation
+  enabled: true
+  
+  # Review triggers
+  triggers:
+    - pull_request
+    - push
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)